### PR TITLE
GUI-2310: Enable multi-select actions on images landing page

### PR DESCRIPTION
**Pull request by kamalgill**
Implements https://eucalyptus.atlassian.net/browse/GUI-2310

### CODE REVIEW

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com//job/user-console-PR-monitor/1610/
Test FAILed.

**Comment by aliceh** [comment]
Retest this please

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com//job/user-console-PR-monitor/1611/
Test PASSed.

File: eucaconsole/static/js/pages/images.js [hunk 1]
```diff
@@ -41,6 +46,57 @@ angular.module('ImagesPage', ['LandingPage', 'EucaConsoleUtils'])
...
+            snapshotIDs.forEach(function (snapshotId) {
+                var url = $scope.imagesUrl;
+                var promise = $http.get(url.replace('_id_', snapshotId));
+                promises.push(promise);
+            });
+            // Wait for all API calls for snapshot Images registered to complete before setting scope variables
+            $q.all(promises).then(function(combinedResults) {
+                combinedResults.forEach(function(oData) {
+                    // NOTE: $q.all hoists results in a data attribute on completion, so use oData.data.results
+                    //       instead of the usual oData.results
```
Comment (dkavanagh) [inline eucaconsole/static/js/pages/images.js]: true when using promises. $http().then() also sees that.

File: eucaconsole/static/js/pages/images.js [hunk 2]
```diff
@@ -41,6 +46,57 @@ angular.module('ImagesPage', ['LandingPage', 'EucaConsoleUtils'])
...
+                    var snapshotImagesMapping = {'snapid': snapshotId, 'images': imagesRegistered};
+                    if (oData.data.results.length > 1) {
+                        $scope.snapshotsWithOtherRegisteredImages.push(snapshotImagesMapping);
+                    } else {
+                        $scope.snapshotsWithoutOtherRegisteredImages.push(snapshotImagesMapping);
+                        $scope.imageSnapshotID = snapshotId;
+                    }
+                });
+                modal.foundation('reveal', 'open');
+            }).catch(function(oData, status) {
```
Comment (dkavanagh) [inline eucaconsole/static/js/pages/images.js]: note, this also changes. See how I did it in GUI-2360 branch (static/js/pages/stack_update.js:349

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com//job/user-console-PR-monitor/1616/
Test PASSed.

**Pull request merged**